x-pack/plugins/observability_solution/slo/kibana.jsonc

```diff
@@ -37,6 +36,7 @@
       "presentationUtil",
       "features",
       "licensing",
+      "toastNotifications",
```

Comment:
🤔 not sure we have a plugin called "toastNotifications"?

Reply:
haha that's funny, let me check